### PR TITLE
Go-site issue 1558 gorule 59

### PR DESCRIPTION
**Pull request by dougli1sqrd**
gorule-0000059 to upgrade gaf 2.1 qualifiers into better relations based on GO Term.

https://github.com/geneontology/go-site/issues/1558

### CODE REVIEW

File: ontobio/io/gafparser.py [hunk 1]
```diff
@@ -144,13 +156,16 @@ def parse_line(self, line):
                     else:
                         logger.info("Detected GAF version {}, so using 2.1".format(version))
                         self.version = self.default_version
+                        # Compute the cell component subclass closure
```
Comment (dustine32) [inline ontobio/io/gafparser.py:159]: @dougli1sqrd Does this here mean the closure will be recalculated on every call of `parse_line` for a 2.1 GAF?

Comment (dougli1sqrd) [reply ontobio/io/gafparser.py:159]: No, this only computes the closure if we are in the headers and once we have detected that we are in gaf 2.1. So this should happen only once per gaf 2.1 file.

Comment (dustine32) [reply ontobio/io/gafparser.py:159]: Ah, right. I just realized that after staring harder. All is good, thanks!

**Review by dustine32 — APPROVED** [review]

**Comment by kltm** [comment]
@dougli1sqrd Happy to see merged.

**Pull request merged**